src/scenes/Activities/scenes/EventHistory.js

```diff
@@ -335,7 +337,7 @@ export default class EventHistory extends Component {
     );
     result[1] = (
       <TableRow key={`txaddr-${txid}`} selected onClick={this.handleClick(txid)} className={isExpanded ? classes.show : classes.hide}>
-        <TransactionHistoryAddress transaction={transaction} className={classes.detailRow} />
```

Comment:
don't remove style, follow the design by adding corresponding styles. You can find the styles in previous commit.

Reply:
comment above